### ISSUE TITLE
Fontawesome dependency hardcoded on css

### ISSUE THREAD

**Issue by asadito** [stevermeister/ngWig#151]
By using ngWig in a project which has a Font Awesome library version newer than 4.4 as a dependency, the ngWig library downgrades that library to version 4.4.

On ng-wig.css file there's an import to Font Awesome library:
`@import url("https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css");`
It should be located on the bower.json file instead as a dependency, so we can manage version conflicts on our projects.

**Comment by stevermeister** [comment]
yes, agree it's not good to have such a hardcode.
but nearest future probably we will skip bower as well (for npm).

**Comment by prismsoundzero** [comment]
I agree with @asadito such hardcoded dependency should be removed. We are using ng-wig in our project and this is causing a problem.
